interface/forms/button_ros/new.php

```diff
@@ -0,0 +1,667 @@
+<?php
+//by Art Eaton
+include_once("../../globals.php");
+include_once("$srcdir/api.inc");
+formHeader("Form: button_ros");
+?>
+
+<form method=post action="<?echo $rootdir;?>/forms/button_ros/save.php?mode=new" name="my_form">
+
+ 
+<?php
+$tempquery = "SELECT date FROM form_encounter WHERE encounter = $encounter";
+ $datequery = sqlStatement($tempquery);
+$encdate = SqlFetchArray($datequery);
+
+$old_date = strtotime($encdate['date']);
+$todays_date = strtotime(date("Ymd"));
+if($old_date+(86400) < $todays_date) {
+echo xlt('It is not permitted to enter a Review of Systems form in an encounter that is dated earlier than 3 days prior to today\'s date.');
+} else {
+?>
+<a href="javascript:top.restoreSession();document.my_form.submit();" class="link_submit">[<?php echo xlt('Save');?>]</a>
+<a href="<?php echo $GLOBALS['form_exit_url']; ?>" class="link_submit"
+ onclick="top.restoreSession()">[<?php echo xlt('Don\'t Save');?>]</a>
+<br>
+<?php echo xlt('You must EDIT the form after saving and reviewing to finalize.');?>
+	
+	<table id='ros'>
+	
+	<TR>
+		<TD> <?php echo xlt('REVIEW of SYSTEMS:');?></TD>
+	</TR>
+	<TR class="alt">
+	<TD>
+	<B><?php echo xlt('GEN:');?></B>
+	</TD>
+	</TR>
+	<TR>
+	<TD>
+  
+		<span class="checkbox-button">
+			<input id="GEN1" name="GEN1" type="checkbox" value="on">
+			<label for="GEN1" > <?php echo xlt('Fatigue');?></label>
+		</span>
+
+		<span class="checkbox-button">
+			<input id="GEN2" name="GEN2" type="checkbox" value="on" >
+			<label for="GEN2"> <?php echo xlt('Fever');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GEN3" name="GEN3" type="checkbox" value="on" >
+			<label for="GEN3"> <?php echo xlt('Chills');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GEN4" name="GEN4" type="checkbox" value="on" >
+			<label for="GEN4"> <?php echo xlt('Night Sweats');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GEN5" name="GEN5" type="checkbox" value="on" >
+			<label for="GEN5"> <?php echo xlt('Recent Weight Gain');?></label>
+		</span>
+		
+		<span class="checkbox-button">
+			<input id="GEN6" name="GEN6" type="checkbox" value="on" >
+			<label for="GEN6"> <?php echo xlt('Recent Weight Loss');?></label>
+		</span>
+		</TD>
+		</TR>
+		<TR>
+		<TD>
+		<?php echo xlt('Note abnormalities:');?> 
+					<TEXTAREA NAME="GEN_notes" ROWS=3 COLS=25 WRAP=SOFT STYLE="width: 2in; height: 0.21in"></TEXTAREA>
+					</TD>
+	</TR>	
+
+		<TR class="alt">
+	<TD>
+	<B><?php echo xlt('EYES:');?> </B>
```

Comment:
I always got shot down for punctuation in the information to be translated it breaks the translation